src/controllers/expenses.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const expensesServices = require('../services/expenses');
+const { isValidData } = require('../utils/expenses');
+
+const getAllExpenses = (req, res) => {
+  const expenses = expensesServices.getAllExpenses();
+
+  res.send(expenses);
+};
+
+const getExpenseById = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expensesServices.getExpenseById(expenseId);
+
+  if (typeof +expenseId !== 'number'
+    || !Number.isInteger(+expenseId)
+    || +expenseId <= 0
+  ) {
```

Comment:
```suggestion
  if (typeof expenseId !== 'number'
    || expenseId <= 0
  ) {
```

Reply:
So if a passed id isn't an integer is it a 404 error?

Reply:
That's what you wrote